apps/projects/app/components/Shared/FilterBar/FilterDropDown.js

```diff
@@ -0,0 +1,163 @@
+import React, { useState } from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { Spring, animated } from 'react-spring'
+import ClickOutHandler from 'react-onclickout'
+import { springs, GU, theme } from '@aragon/ui'
+import FilterButton from './FilterButton'
+import { IconMore, IconDropArrow } from '../../../assets'
+
+const FilterDropDown = ({ caption, children, enabled, type }) => {
+  const [ opened, setOpened ] = useState(false)
+
+  const handleClose = () => {}
+
+  const handleClickOut = () => setOpened(false)
+
+  const handleBaseButtonClick = () => {
+    if (enabled) setOpened(!opened)
+  }
+
+  return (
+    <ClickOutHandler onClickOut={handleClickOut}>
+      <Spring
+        config={springs.smooth}
+        to={{ openProgress: Number(opened) }}
+        native
+      >
+        {({ openProgress }) =>
+          type === 'overflow' ? (
```

Comment:
I'd rather we kept `FilterDropDown` and `FilterOverflow` as separate components, rather than passing this `type` property in. We could move the `ClickOutHandler` and `Spring` stuff to some other component, if we wanted. Is that the only reason that they're combined right now?

Reply:
They started as one component and diverged to the current point - which is the point they can be properly separated. Will to a special ticket.